provisioning/service/tests/Config/AttestationMechanismTests.cs

```diff
@@ -1,127 +0,0 @@
-﻿// Copyright (c) Microsoft. All rights reserved.
-// Licensed under the MIT license. See LICENSE file in the project root for full license information.
-
-using System;
-using System.Text;
-using System.Text.Json;
-using Microsoft.VisualStudio.TestTools.UnitTesting;
-
-namespace Microsoft.Azure.Devices.Provisioning.Service.Test
-{
-    [TestClass]
-    [TestCategory("Unit")]
-    public class AttestationMechanismTests
```

Comment:
Most of these tests were validating the json constructor logic... which we no longer have... so, I deleted them.